src/ansiblelint/_prerun.py

```diff
@@ -29,30 +29,62 @@
 """
 
 
-def check_ansible_presence() -> None:
-    """Assures we stop execution if Ansible is missing."""
+def check_ansible_presence(exit_on_error=False) -> Tuple[str, str]:
+    """Assures we stop execution if Ansible is missing or outdated.
+
+    Returne found version and an optional exception if something wrong
+    was detected.
+    """
+    err = ""
     failed = False
-    try:
-        # pylint: disable=import-outside-toplevel
-        from ansible import release
-
-        if version.parse(release.__version__) <= version.parse(ANSIBLE_MIN_VERSION):
-            failed = True
-    except (ImportError, ModuleNotFoundError, UnboundLocalError) as e:
-        failed = True
-        __version__ = "none"
-        print(e, file=sys.stderr)
-    if failed:
-        print(
-            "FATAL: ansible-lint requires a version of Ansible package"
-            " >= %s, but %s was found. "
-            "Please install a compatible version using the same python interpreter. See "
-            "https://docs.ansible.com/ansible/latest/installation_guide"
-            "/intro_installation.html#installing-ansible-with-pip"
-            % (ANSIBLE_MIN_VERSION, __version__),
-            file=sys.stderr,
-        )
+    ansible_cli_version = ""
+    result = subprocess.run(
+        args=["ansible", "--version"],
+        stdout=subprocess.PIPE,
+        universal_newlines=True,
+        check=False,
+    )
+    if result.returncode != 0:
+        err = "FATAL: Unable to retrieve ansible cli version: %s" % result.stdout
```

Comment:
How about moving the main logic into another function which always returns an error; then you can simplify this function like:
```suggestion
        return ansible_cli_version, "FATAL: Unable to retrieve ansible cli version: %s" % result.stdout
```
Then you don't have to indent the remainder of the function.